tests/unit/ls

```diff
@@ -0,0 +1,146 @@
+import crawl from '../../src/crawl.js';
```

Comment:
redundant file?

Reply:
Oops. This is one of the things I should have caught during self-review. Will delete.

Reply:
removed a814beb